packages/components/src/components/spinner/spinner.stories.ts

```diff
@@ -1,75 +1,69 @@
 import '../../solid-components';
 import { html } from 'lit-html';
-import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
+import { storybookDefaults } from '../../../scripts/storybook/helper';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes, args, parameters } = storybookDefaults('sd-spinner');
-const { generateTemplate } = storybookTemplate('sd-spinner');
+
+/**
+ * Visual indicator with a looping animation that shows loading is in process.
+ */
 
 export default {
   title: 'Components/sd-spinner',
   component: 'sd-spinner',
+  tags: ['!dev'],
   args,
   argTypes,
   parameters: { ...parameters },
   decorators: [withActions] as any
 };
 
-/**
- * Default: This shows sd-spinner in its default state.
- */
-
 export const Default = {
-  render: (args: any) => {
-    return generateTemplate({ args });
-  }
+  name: 'Default',
+  render: () => html`<sd-spinner></sd-spinner>`
 };
 
 /**
- * Use the `color` attribute to change the color of the spinner and correspond (currentColor) with the parents color.
+ * Offers `color` variants for alternate experiences:
+ * - `currentColor` (default): used to inherit the color from the parent element
+ * - `primary`: used on light backgrounds
+ * - `white`: used on primary backgrounds
  */
 
-export const Color = {
+export const Variants = {
+  name: 'Variants',
   parameters: { controls: { exclude: 'color' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'color' }
-      },
-      options: {
-        templateBackgrounds: {
-          alternate: 'x',
-          colors: ['', 'rgb(var(--sd-color-primary, 0 53 142))', '#F6F6F6']
-        }
-      },
-      args
-    });
-  }
+  render: () =>
+    html` <div class="flex flex-row items-center gap-12">
+      <div class="bg-neutral-100 p-4">
```

Comment:
Has it been agreed with design that `bg-neutral` should be added to `currentColor`?

Reply:
Discussed with Design and agreed to remove the background